packages/itwinui-css/src/table/table.scss

```diff
@@ -196,6 +196,11 @@
   &.iui-zebra-striping .iui-row:nth-child(even):not(.iui-selected) .iui-cell {
     @include themed {
       background-color: rgba(t(iui-color-foreground-body-rgb), 0.02);
```

Comment:
looks like this `background-color` gets overridden by the `background` in next line so it shouldn't be needed

```suggestion
```

Reply:
Oh wow, forgot to remove old styling. Good catch!